pytest_fixtures/component/provision_pxe.py

```diff
@@ -85,8 +86,7 @@ def module_provisioning_rhel_content(
     ).create()
 
     host_root_pass = settings.provisioning.host_root_password
-    pxe_loader = "PXELinux BIOS"  # TODO: Make this a fixture parameter
-
+    pxe_loader = "PXELinux BIOS" if provisioning_host.vm_firmware == 'bios' else "Grub2 UEFI"
```

Comment:
either use single/double quotes, single quotes are preferred
```suggestion
    pxe_loader = 'PXELinux BIOS' if provisioning_host.vm_firmware == 'bios' else 'Grub2 UEFI'
```